### PR TITLE
PT-56

### PR DESCRIPTION
**Pull request by ItsFaizan**
# Navbar - PT-35

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -10,7 +12,11 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body><Navbar></Navbar>{children}</body>
+      <head>
+      <link rel="preconnect" href="https://stijndv.com" />
+      <link rel="stylesheet" href="https://stijndv.com/fonts/Eudoxus-Sans.css" />
```
Comment (fuddin95) [inline src/app/layout.tsx:17]: Why do we need these?

File: src/app/static/Inter-Black.ttf
Comment (fuddin95) [inline src/app/static/Inter-Black.ttf:1]: Better place for static files is under public folder or move static outside the app folder

File: src/app/layout.tsx [hunk 2]
```diff
@@ -10,7 +12,11 @@ export default function RootLayout({
...
+    </head>
+    <body><Navbar></Navbar>{children}</body>
```
Comment (fuddin95) [inline src/app/layout.tsx:19]: Change it to `<Navbar />`

File: src/app/layout.tsx (hunk 1 above)
Comment (ItsFaizan) [reply src/app/layout.tsx:17]: It is required so that we can use global.css and remove that extra margin and padding

File: src/app/static/Inter-Black.ttf
Comment (ItsFaizan) [reply src/app/static/Inter-Black.ttf:1]: ok

Comment (ItsFaizan) [reply src/app/static/Inter-Black.ttf:1]: Done

**Review by fuddin95 — APPROVED** [review]

**Pull request merged**